frontend/package.json

```diff
@@ -76,7 +76,7 @@
     "fbjs": "^3.0.4",
     "fetchify": "0.0.2",
     "file-loader": "6.2.0",
-    "flagsmith": "3.18.0",
```

Comment:
Lol, this is a _big_ jump! (I know it's mostly because of the major version wrangling we had to do, but it's just amusing).

Reply:
we can use  `<FlagsmithProvider>` now